float64 max_duration
float64 min_duration
float64 avg_duration
float64 tot_duration
float64 cur_duration
uint32  count
